thing-url-adapter.js

```diff
@@ -264,8 +263,27 @@ class ThingURLDevice extends Device {
     if (this.closing) {
       return;
     }
+    // if we have an entry for the wss url stub in jwt_auth
+    // add auth to query string
+    let auth = '';
+
+    for (const i in this.adapter.authData) {
+      if (this.wsUrl.includes(i)) {
+        switch (this.adapter.authData[i].method) { // 0 is the method - jwt etc
```

Comment:
nit: the comment here can be removed now

Reply:
done